source/configuring-ember/configuring-your-app.md

```diff
@@ -2,7 +2,7 @@ Ember CLI ships with support for managing your application's environment. Ember
 
 The ENV object contains three important keys:
 
-  - `EmberENV` can be used to define Ember feature flags (see the [Feature Flags guide](http://emberjs.com/guides/configuring-ember/feature-flags/)).
+  - `EmberENV` can be used to define Ember feature flags (see the [Feature Flags guide](http://guides.emberjs.com/v2.0.0/configuring-ember/feature-flags/)).
```

Comment:
This should be a relative URL, so that each future version links to its own copy.